q2_quality_control/decontam.py

```diff
@@ -58,27 +71,23 @@ def _check_column_inputs(table, metadata, method, freq_concentration_column,
                          'metadata:\n'
                          + str(', '.join(no_info)))
     if method == 'prevalence':
-        _check_column_inputs_helper(table, metadata,
-                                    prev_control_column,
-                                    prev_control_indicator)
-
+        if freq_concentration_column is not None:
+            print("Warning: --p-freq-concentration-column given, but" +
+                  " disregarded")
+        _check_prev_inputs(table, metadata, prev_control_column,
+                           prev_control_indicator, method)
     elif method == 'frequency':
-        if freq_concentration_column not in metadata.columns:
-            raise ValueError('Frequency column not found, please '
-                             'select from:\n'
-                             + str(', '.join(metadata.columns)))
-        else:
-            print("All appropriate inputs are found")
+        if prev_control_column is not None:
+            print("Warning: --prev-control-column given, but" +
```

Comment:
Same as above

Reply:
addressed in commit [f31210b](https://github.com/qiime2/q2-quality-control/pull/88/commits/f31210b1b4524e53c8545435910ecb106d40fc8d)